suite-native/module-onboarding/src/components/Camera.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect, useState } from 'react';
+import { Camera as RNCamera, useCameraDevices } from 'react-native-vision-camera';
+import { Dimensions, Linking, View } from 'react-native';
+
+import { BarcodeFormat, useScanBarcodes } from 'vision-camera-code-scanner';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+export interface CameraProps {
+    onResult: (value?: string) => void;
+}
+
+const cameraWrapperStyle = prepareNativeStyle<{ isCameraAllowed: boolean }>(
+    (utils, { isCameraAllowed }) => ({
+        height: 329,
```

Comment:
do we really need fixed heigh here?

Reply:
According to [Figma](https://www.figma.com/file/Z6AGVUmKQzLNtDozFamW7f/branch/Ad5LG1FdIjup5NjyG8abHY/s2-Mobile?node-id=2793%3A4384).